reference-architecture/rhv-ansible/example/ocp-vars.yaml.beta

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
That comes from trying to track versions of three products without making the file names way too long.
Beta was for RHEL 7.5 beta, which required hitting beta channels in OS (but I left off the effort because it was more work to make 7.5 work than to make 7.4 images talk to RHV 4.2)